com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1241,14 +1244,16 @@ void Resize(HDCamera hdCamera)
                 {
                     LightLoopReleaseResolutionDependentBuffers();
                     m_DbufferManager.ReleaseResolutionDependentBuffers();
+                    m_SharedRTManager.DisposeFullScreenDebugBuffer();
                     m_SharedRTManager.DisposeCoarseStencilBuffer();
                 }
 
                 LightLoopAllocResolutionDependentBuffers(hdCamera, m_MaxCameraWidth, m_MaxCameraHeight);
+                m_SharedRTManager.AllocateFullScreenDebugBuffer(m_MaxCameraWidth, m_MaxCameraHeight, m_MaxViewCount);
                 if (!m_EnableRenderGraph)
                 {
                     m_DbufferManager.AllocResolutionDependentBuffers(m_MaxCameraWidth, m_MaxCameraHeight);
-                    m_SharedRTManager.AllocateCoarseStencilBuffer(m_MaxCameraWidth, m_MaxCameraHeight, hdCamera.viewCount);
+                    m_SharedRTManager.AllocateCoarseStencilBuffer(m_MaxCameraWidth, m_MaxCameraHeight, m_MaxViewCount);
```

Comment:
Why this change ?m_MaxViewCount

Reply:
Because that could lead to an incorrect allocation in some cases where you change resolution and view count in a specific order (because no reallocation is done when hdCamera.viewCount is inferior to m_MaxViewCount)

Reply:
@fabien-unity isn't this a problem for VR?
@adrien-de-tocqueville so it is a bug unrelated to this PR?

Reply:
Yes it's unrelated, i just saw it because I had to copy the line

Reply:
> @fabien-unity isn't this a problem for VR?
all good. this will avoid issues when switching between VR and non-VR modes.